src/interceptors/ClientRequest/utils/getIncomingMessageBody.ts

```diff
@@ -1,13 +1,13 @@
 import { IncomingMessage } from 'http'
 
-export function getIncomingMessageBody(res: IncomingMessage): Promise<string> {
-  let responseBody = ''
+export function getIncomingMessageBody(res: IncomingMessage): Promise<Buffer> {
+  const bufs: Buffer[] = [];
 
   return new Promise((resolve, reject) => {
     res.once('error', reject)
-    res.on('data', (chunk) => (responseBody += chunk))
```

Comment:
Are there any implications if we handle this `responseBody` and `chunk` handling as Buffer here instead of adjusting all the other places?